Xamarin.Forms.Platform.iOS/Renderers/EntryRenderer.cs

```diff
@@ -488,9 +493,16 @@ void SetSelectionLengthFromRenderer(int selectionLength)
 			}
 		}
 
-        void UpdateIsReadOnly()
-        {
-            Control.UserInteractionEnabled = !Element.IsReadOnly;
-        }
-    }
+		void UpdateIsReadOnly()
+		{
+			Control.UserInteractionEnabled = !Element.IsReadOnly;
+		}
+
+		void UpdateClearButtonVisibility(bool isInitializating)
+		{
+			bool showClearButton = Element.ClearButtonMode == InputViewClearButtonMode.WhileEditing;
+			if ((isInitializating && showClearButton) || !isInitializating)
```

Comment:
Thanks for the review! 
I think the logic is correct albeit the condition could be simplified to: 
`if(!isInitializing || showClearButon)`

Here's the logic I implemented:
My goal is to call the native API (`Control.ClearButtonMode`) when it's really needed, which is in two situations:
- either XF `ClearButtomMode` property has changed, and in this case native API must be set to reflect the XF property change
- or the renderer is initializing and the XF property is set to `WhileEditing` so native must be called to reflect the mode (on iOS the default mode for UITexField is not to show the clear button)

Further more, Instead of `isInitializing` and negation, I changed it to `forceUpdate`, I pushed the changes, it should be clearer.

Or am I missing something, what's the bug you're seeing?

Reply:
Ah, I again thought that result of the if statement was the value set. My bad.

Reply:
No worries. Let me know if you think there's something which could improve readability.